doc/rippled-example.cfg

```diff
@@ -842,7 +842,7 @@ port_wss_admin
 port = 5005
 ip = 127.0.0.1
 admin = 127.0.0.1
-protocol = https
+protocol = http
```

Comment:
Doesn't uniport allow us to specify both secured and unsecured protocols, and decides by examining the message?

I think the rippled client code only looks at the first option here, but would